core-java-modules/core-java-streams-3/src/test/java/com/baeldung/streams/bigdecimals/AddNumbersUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.streams.bigdecimals;
+
+import static org.junit.Assert.assertEquals;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import org.junit.Test;
+
+public class AddNumbersUnitTest {
+
+    @Test
+    public void givenIntStream_whenSum_thenResultIsCorrect() {
```

Comment:
Note: I've renamed this method - the then doesn't describe what the test does, it describes the required outcome.

Reply:
Sounds good, thanks